app/assets/stylesheets/signin_form.scss

```diff
@@ -0,0 +1,31 @@
+a {
+  color: #A486B6;
+}
+
+a:hover {
+  color: #532E69;
+}
+
+body {
+  background-color: #fff;
+  padding-top: 30px;
+}
+
+.container {
+  -webkit-border-radius: 5px;
+     -moz-border-radius: 5px;
+          border-radius: 5px;
+  -webkit-box-shadow: 0 1px 2px rgba(0,0,0,.05);
+     -moz-box-shadow: 0 1px 2px rgba(0,0,0,.05);
+          box-shadow: 0 1px 2px rgba(0,0,0,.05);
+  background-color: #fff;
```

Comment:
Please update this to be upcased, I can see you are using upcased color values...